### PR TITLE
use simple <get> task to get bcel, and asm

### PR DESCRIPTION
**Pull request by mebigfatguy**

### CODE REVIEW

**Comment by iloveeclipse** [comment]
What is the reason for the change? Are there any troubles with the current approach? For the proposed change: does it mean that files are downloaded on every execution?

**Comment by mebigfatguy** [comment]
i had multiple problems with the current approach, one that juan seemed to fix, as seen here

https://github.com/spotbugs/spotbugs/issues/1

the next one, which remains, as

/home/dave/dev/spotbugs/findbugs/build.xml:1295: org.mozilla.javascript.EcmaError: ReferenceError: "print" is not defined.
	at org.mozilla.javascript.ScriptRuntime.constructError(ScriptRuntime.java:3951)



We can remove the tasks from the build target, there's no reason to run them all the time, if you like

There's also a 

usetimestamp="true"

attribute which only downloads if a version is newer., which we can add.

i'm fine with leaving the current as is, if someone can fix it, it just seemed to be broken.

**Comment by jsotuyod** [comment]
Bare in mind `usetimestamp` relies on local modification time of the file. If you `touch` an empty file with that name, it will still consider it up to date. Nothing too serious, but still to consider.

The current download code however, is giving me issues when trying to get Travis to run using Java9.

```
BUILD FAILED
/home/travis/build/spotbugs/spotbugs/build.xml:37: The following error occurred while executing this line:
/home/travis/build/spotbugs/spotbugs/findbugs/build.xml:1295: Unable to load a script engine manager (org.apache.bsf.BSFManager or javax.script.ScriptEngineManager)
```

https://travis-ci.org/spotbugs/spotbugs/jobs/173798833#L2064-L2066

On Java 8 it runs 'though... https://travis-ci.org/spotbugs/spotbugs/jobs/173798832

File: findbugs/build.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <!--
             The local.properties properties file contains the location of eclipsePlugin.dir
 
-            This value is likely to be different for each checkout of the plugin,
+            This valueg is likely to be different for each checkout of the plugin,
```
Comment (henrik242) [inline findbugs/build.xml]: What's this `valueg` thing you are talking about? :)

**Comment by jsotuyod** [comment]
I tracked down my build issue. ant `<script>` task is broken with modular jdk9 due to a known ant issue: https://bz.apache.org/bugzilla/show_bug.cgi?id=58271 

This issue is due to change in behaviour of `ClassLoader.getResource` for resources in named modules, and is fixed on ant 1.9.6.

I am manually installing ant 1.9.7 on Travis, but I approve of this change once the typo marked by @henrik242.

Ant sucks at dependency management anyway, but we will get to discussing that when we attack #4

**Comment by jsotuyod** [comment]
I'll revert the download of Ant 1.9.7 on Travis based on this to speed up build / make sure build works on all Java versions for everyone.

**Pull request merged**